src/main/CommandLine.cpp

```diff
@@ -1041,30 +1041,109 @@ runReportLastHistoryCheckpoint(CommandLineArgs const& args)
         });
 }
 
+std::vector<std::pair<uint32_t, uint32_t>>
+parseSimulateSleepPerOp(std::vector<std::string> const& inputs)
+{
+    // The input must be of the form percentage/duration
+    // where
+    //     * 0 <= percentage <= 100
+    //     * 0 <= duration
+    //     * percentage for slow + percentage for medium + percentage for fast =
+    //     100.
+    //     * duration for slow >= duration for medium >= duration for fast.
+    if (std::any_of(inputs.begin(), inputs.end(),
```

Comment:
I don't think this function should care about slow/medium/fast: it's just a vector of `percentage/duration` that needs to be transformed

Reply:
Removed the slow/medium/fast stuff.